src/assets/yii2-dynamic-form.js

```diff
@@ -115,7 +115,7 @@
         var count = _count($elem, widgetOptions);
 
         if (count < widgetOptions.limit) {
-            $toclone = widgetOptions.template;
+            $toclone = $(widgetOptions.template);
```

Comment:
If you log $toClone you would find its a string rather than jQuery Object. Thats why I have explicitly create the jQuery object